testinfra/vars/app-staging.yml

```diff
@@ -22,8 +22,8 @@ apache_source_log: /var/log/apache2/source-error.log
 apache_allow_from: all
 
 dns_server: "8.8.8.8"
-mon_ip: 10.0.1.3
-app_ip: 10.0.1.2
+mon_ip: 172.16.1.203
+app_ip: 172.16.1.69
```

Comment:
ahhhhhhhh :hankey: I think i should fix that functionality to use a file that is ignored from git. Its too easy to make a mistake like I just did.... :cry: